terraform/modules/tf_stream_alert_globals/variables.tf

```diff
@@ -2,6 +2,10 @@ variable "account_id" {}
 
 variable "prefix" {}
 
+variable "classifier_use_prefix" {
```

Comment:
I like the idea of using levers to accommodate the migration. This way you can perform the whole migration by only changing variables.

That said, this won't quite work, as Terraform will still destroy the old SQS resource while creating the new one. Take a look at my [previous comment](https://github.com/airbnb/streamalert/pull/955#issuecomment-509806439)

Reply:
It might be reasonable to just not upgrade an in-place environment and only add a prefix to new installations. We're defaulting to false here on the prefix so unless you explicitly decide to add a prefix to an existing environment this code is non-destructive. I'm not sure we understand the structure of your code well enough to build the logic to replace an existing queue correctly.

Reply:
No worry, I got you guys. Going to punch in a fix to the prefix problem today/tomorrow. It's been bothering me for months too

Stay tuned...

Reply:
Thanks @Ryxias